deploy/charts/trust-manager/values.yaml

```diff
@@ -182,6 +182,13 @@ app:
   # The verbosity of trust-manager logging. This takes a value from 1-5, with the higher value being more verbose.
   logLevel: 1
 
+  # Leader Election Configuration
```

Comment:
```suggestion
```

Nit: Seems a bit superfluous.

Reply:
I agree with Erik's suggestions here so I've added them to a commit and pushed it 👍